src/bindings/python/src/pyopenvino/core/compiled_model.cpp

```diff
@@ -40,13 +40,29 @@ void regclass_CompiledModel(py::module m) {
 
     cls.def("export_model", &ov::CompiledModel::export_model, py::arg("model_stream"));
 
+    cls.def(
+        "set_property",
+        [](ov::CompiledModel& self, const std::map<std::string, std::string>& properties) {
+            self.set_property({properties.begin(), properties.end()});
+        },
+        py::arg("properties"));
+
+    // todo: remove after Accuracy Checker migration to set/get_property API
     cls.def(
         "get_config",
         [](ov::CompiledModel& self, const std::string& name) -> py::object {
             return Common::from_ov_any(self.get_property(name)).as<py::object>();
         },
         py::arg("name"));
 
+    cls.def(
+        "get_property",
+        [](ov::CompiledModel& self, const std::string& name) -> py::object {
+            return Common::from_ov_any(self.get_property(name)).as<py::object>();
+        },
```

Comment:
```suggestion
        "get_property",
        [](ov::CompiledModel& self, const std::string& name) {
            return Common::from_ov_any(self.get_property(name));
        },
```
`from_ov_any` is returning actual simple types of different classes (like int, float, vectors, map, tuples) which would be automatically converted to Python object by pybind's mechanism. If not please specify case/cases and extend common helper if possible.

Reply:
removed